gravityflow/gravityflowwoocommerce
# Address the get used coupon field mapping bug

re:[HS#14055](https://secure.helpscout.net/conversation/1216357568/14055/)

This PR fixes an issue where the "Coupon Code Used" cannot be mapped in the WooCommerce settings.

## Testing instructions
1. See the doc about how to map the "Coupon Code Used" field.
2. You can use the testing form Subrato attached to that ticket.
3. Confirm in the `master` branch the mapping didn't work, switch to this branch should resolve it.
